assets/ar/factuality_disinformation_harmful_content/subjectivity/ThatiARSubjectivity_Sonet_ZeroShot_en.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from llmebench.datasets import ThatiARDataset
+from llmebench.models import AnthropicModel
+from llmebench.tasks import SubjectivityTask
+
+
+def metadata():
+    return {
+        "author": "Arabic Language Technologies, QCRI, HBKU",
+        "model": "claude-3-5-sonnet-20240620",
+        "description": "Anthropic model",
+        "scores": {},
+    }
+
+
+def config():
+    system_msg = "AI assistant specialized in classifying news article sentences into subjective or objective. A subjective sentence expresses personal opinions, feelings, or beliefs, while an objective sentence presents facts, data, or unbiased information."
+    return {
+        "dataset": ThatiARDataset,
+        "task": SubjectivityTask,
+        "model": AnthropicModel,
+        "model_args": {
+            "class_labels": ["SUBJ", "OBJ"],
+            "max_tries": 30,
+            "system": system_msg,
+        },
+    }
+
+
+def prompt(input_sample):
+
+    prompt = f"""
+        Classify the following Arabic 'sentence' as subjective or objective. Provide only the label. 
+        Provide your response in the following JSON format: {{"label": "your label"}}. 
+        Please provide JSON output only. No additional text.
```

Comment:
Why json? I think adding this part made the prompt too long.

Reply:
For original implementation, we also asked to provide explanation, score etc. For which json structure is important to get them right.